src/cli/tc.rs

```diff
@@ -61,51 +61,94 @@ pub fn run() -> Result<()> {
 }
 
 pub async fn init(file_path: &str) -> Result<()> {
+  let folder_exists = fs::metadata(file_path).is_ok();
+
+  if !folder_exists {
+    let confirm = Confirm::new(&format!("Do you want to create the folder {}?", file_path))
+      .with_default(false)
+      .prompt();
+
+    match confirm {
+      Ok(true) => fs::create_dir(file_path)?,
+      Ok(false) => (),
+      Err(e) => return Err(e.into()),
+    };
+  }
+
   let tailcallrc: resource::Resource<str> = resource_str!("examples/.tailcallrc.graphql");
 
-  let ans = Confirm::new("Do you want to add a file to the project?")
-    .with_default(false)
-    .prompt();
-
-  match ans {
-    Ok(true) => {
-      let file_name = inquire::Text::new("Enter the file name:")
-        .with_default(".graphql")
-        .prompt()
-        .unwrap_or_else(|_| String::from(".graphql"));
-
-      let file_name = format!("{}.graphql", file_name.strip_suffix(".graphql").unwrap_or(&file_name));
-
-      let confirm = Confirm::new(&format!("Do you want to create the file {}?", file_name))
-        .with_default(false)
-        .prompt();
-
-      match confirm {
-        Ok(true) => {
-          fs::write(format!("{}/{}", file_path, &file_name), "")?;
-
-          let graphqlrc = format!(
-            r#"|schema:
-               |- './{}'
-               |- './.tailcallrc.graphql'
-          "#,
-            &file_name
-          )
-          .strip_margin();
-          fs::write(format!("{}/.graphqlrc.yml", file_path), graphqlrc)?;
+  let file_name = ".tailcallrc.graphql";
+  let yml_file_name = ".graphqlrc.yml";
+  let yml_exists = fs::metadata(format!("{}/{}", file_path, yml_file_name)).is_ok();
+
+  if yml_exists {
+    // what should we do in this case?
```

Comment:
@meskill in your pseudo-code on [this comment](https://github.com/tailcallhq/tailcall/issues/698#issuecomment-1865172783), you mentioned

> `  tc_rc = gq_rc.get_tc_rc();  `

to be on this line, but I don't know what that's supposed to do.
IMO, we don't need it; if that's the case, please confirm, and I'll change the condition to `!yml_exists` and remove the `else` block

Reply:
it's supposed to be getting the path to tailcall graphql file from the yml file in case it has some non-default placement